guides/doc-Administering_Red_Hat_Satellite/master.adoc

```diff
@@ -17,6 +17,10 @@ include::common/assembly_using-satellite-ansible-content-collections.adoc[levelo
 
 include::topics/Users_and_Roles.adoc[]
 
+ifdef::katello,satellite[]
+include::common/assembly_configuring-puppet-in-satellite.adoc[leveloffset=+1]
```

Comment:
@maximiliankolb can you confirm the scenarios for this? 
Should be all except orcharhino maybe? 
This is config management rather than content management isn't it, so it is valid for foreman-el?

Reply:
Waiting for comment.